distribution/deps.bzl

```diff
@@ -21,5 +21,5 @@ def vaticle_bazel_distribution():
     git_repository(
         name = "vaticle_bazel_distribution",
         remote = "https://github.com/vaticle/bazel-distribution",
-        commit = "cdf8f3e3e009b3690d0f467b1ecc5b4f122a74fa" # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_bazel_distribution
+        commit = "16f85fa14818c61bd0ed58553ab5d63f2039985d" # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_bazel_distribution
```

Comment:
We also update to the latest `bazel-distribution`, where a similar fix was just written for `assemble_crate`.

Reply:
I've taken a look at the Semantic Versioning specification (https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions). This should comply with any crate name where the name is proceeded by a valid semantic version.